src/controllers/usersurveyresponse.controller.ts

```diff
@@ -0,0 +1,30 @@
+import { getRepository } from 'typeorm';
+import { NextFunction, Request, Response } from 'express';
+
+import UserSurveyResponse from '../entity/UserSurveyResponse';
+import Answer from '../entity/Answer';
+import Choice from '../entity/Choice';
+
+export const createSurveyResponse = async (
+    req: Request,
+    res: Response
+): Promise<Response> => {
+    const responseFromReq = {
+        userId: req.body.user_id,
+        surveyId: req.body.survey_id,
+        completed: true
+    };
+    try {
+        const newResponse: UserSurveyResponse = getRepository(UserSurveyResponse).create(responseFromReq);
+        const savedSurvey: UserSurveyResponse = await getRepository(UserSurveyResponse).save(newResponse);
+        req.body.answers.forEach(async (answer: Answer) => {
+            answer.userSurveyResponseId = savedSurvey.id;
+            const newAnswer: Answer = await getRepository(Answer).create(answer);
```

Comment:
Ogólnie kod jest git moim zdaniem 👍 
Ale musze sie do czegoś przyczepić :) a więc tak:
Ogólnie jest taka zasada eslint'a żeby nie reassignować zmiennych po których się iteruje (w tym przypadku answer)
https://eslint.org/docs/rules/no-param-reassign
Dlatego proponuje poniższe rozwiązanie:
```suggestion
            const newAnswer: Answer = await getRepository(Answer).create({
                ...answer,
                userSurveyResponseId: savedSurvey.id
            });
```
Ogólnie zainstaluj sobie eslinta (on juz jest skonfigurowany w tym projekcie) to będą Ci się pokazywać podpowiedzi właśnie z takimi regułami które pomagają utrzymać kod w czystości.
![image](https://user-images.githubusercontent.com/21139344/99901435-f6219c80-2cb6-11eb-93e8-9283e869cbce.png)
W Vs Code eslint znajdziesz w "extensions" do zainstalowania 

A i jeszcze też zainstaluj sobie taki drugi popularny extension do formatowania kodu "Prettier" bo ustawiłem też w configu tego prettiera space indent na 2 spacje a Tobie z defaultu na 4 zrobiło. No i też Prettier ładnie zawija przydługawe wyrażenia na kilka linijek autoatycznie